perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -2,9 +2,11 @@
 # Running test only for when PR contains execute-perf-test label
 curl https://api.github.com/repos/GoogleCloudPlatform/gcsfuse/pulls/$KOKORO_GITHUB_PULL_REQUEST_NUMBER >> pr.json
 perfTest=$(cat pr.json | grep "execute-perf-test")
+integrationTests=$(cat pr.json | grep "execute-integration-tests")
 rm pr.json
 perfTestStr="$perfTest"
-if [[ "$perfTestStr" != *"execute-perf-test"* ]]
+integrationTestsStr="$integrationTests"
+if [[ "$perfTestStr" != *"execute-perf-test"*  &&  "$integrationTestsStr" != *"execute-integration-tests"* ]]
```

Comment:
Since `execute-perf-test` and `execute-integration-tests` string literals appear so many places, better to declare them as readonly constants at the top.. 
e.g.
```
readonly EXECUTE_PERF_TEST_TAG="execute-perf-test"
readonly EXECUTE_INTEGRATION_TEST_TAG="execute-integration-tests"
```